src/matter/cluster/BooleanStateCluster.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalEvent, EventPriority, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { TlvField, TlvBoolean, MatterApplicationClusterSpecificationV1_0 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Illuminance Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 1.7
+ */
+export const BooleanStateCluster = Cluster({
+    id: 0x0045,
+    name: "BooleanState",
+    revision: 1,
+    features: { }, // no features
```

Comment:
nit: features is optional so you don't need to define this parameter is there are no features